AaxDecrypter/UnencryptedAudiobookDownloader.cs

```diff
@@ -21,7 +21,7 @@ public UnencryptedAudiobookDownloader(string outFileName, string cacheDirectory,
 				Name = "Download Mp3 Audiobook",
 
 				["Step 1: Get Mp3 Metadata"] = Step1_GetMetadata,
-				["Step 2: Download Audiobook"] = Step2_DownloadAudiobook,
+				["Step 2: Download Audiobook"] = Step2_DownloadAudiobookAsSingleFile,
 				["Step 3: Create Cue"] = Step3_CreateCue,
```

Comment:
As I think about this, a CUE file doesn't make sense if you're splitting the audio at cue boundaries.

Reply:
Agreed. And this is looking more and more like we can ditch the 'steps' paradigm

```
step 1: metadata
step 2:
    if multi
        split
    else
        process as single
        create cue
```